backend/src/models/Appointment.ts

```diff
@@ -7,18 +7,21 @@ interface AppointmentAttributes {
   medic_id: number;
   start_at: Date;
   end_at: Date;
-  symptoms?: string;
-  diagnostic?: string;
+  symptoms: string;
+  diagnostic: string;
```

Comment:
se dan al final de la cita 
```suggestion
  symptoms?: string;
  diagnostic?: string;
```